src/SFML/Window/OSX/HIDInputManager.mm

```diff
@@ -288,14 +734,14 @@ void keyboardChanged(CFNotificationCenterRef, void* observer, CFStringRef, const
 {
     ////////////////////////////////////////////////////////////
     // The purpose of this function is to initialize m_keys so we can get
-    // the associate IOHIDElementRef with a sf::Keyboard::Scancode
+    // the associate IOHIDElementRef with a Keyboard::Scancode
```

Comment:
Doesn't really matter, but the namespace could remain here

Reply:
Done :+1: